packages/docs/fluent-editor/demos/flow-chart.vue

```
@@ -0,0 +1,55 @@
+const editorRef = ref<HTMLElement>()
+
+const TOOLBAR_CONFIG = [
+  [{ header: [] }],
+  ['bold', 'italic', 'underline', 'link'],
+  [{ align: [] }, { list: 'ordered' }, { list: 'bullet' }, { list: 'check' }],
+  ['clean'],
+  ['flow-chart'],
+]
+
+onMounted(() => {
+    Promise.all([
```

ESLint

[error] Expected indentation of 2 spaces but found 4.

(style/indent)